src/plugin/connector/__init__.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import requests
+
+from spaceone.core.connector import BaseConnector
+
+__all__ = ["RequestConnector"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RequestConnector(BaseConnector):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    @staticmethod
+    def send_request(url, headers, body=dict, method="GET", page=None, per_page=30):
+        try:
+            if page:
+                for response in RequestConnector.page_nation(
+                    url, headers, body, per_page, page
+                ):
+                    yield response
+            else:
+                response = requests.get(url, headers=headers).json()
+                yield response
+        except Exception as e:
+            _LOGGER.error(f"Request Error: {e}")
+            raise e
+
+    @staticmethod
+    def page_nation(url, headers, body, per_page, page):
```

Comment:
nitpick: In my opinion, "page nation" doesn't seem to be the correct expression; "pagination" appears to be more accurate.